From 166687bbe25d20f5409ba9cdf89be9caa4404666 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:36:46 -0400
Subject: [block] null_blk: fix name and description of 'queue_mode' module parameter

Message-id: <1402670238-13916-119-git-send-email-snitzer@redhat.com>
Patchwork-id: 83849
O-Subject: [RHEL7.1 PATCH 118/150] null_blk: fix name and description of 'queue_mode' module parameter
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit 54ae81cd5a20d23fdc7906a59d9018e1a760cadc
Author: Mike Snitzer <snitzer@redhat.com>
Date:   Wed Jun 11 17:13:50 2014 -0400

    null_blk: fix name and description of 'queue_mode' module parameter

    'use_mq' is not the name of the module parameter, 'queue_mode' is.

    Signed-off-by: Mike Snitzer <snitzer@redhat.com>
    Signed-off-by: Jens Axboe <axboe@fb.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/block/null_blk.c b/drivers/block/null_blk.c
index d2aa4a1..98bb1f0 100644
--- a/drivers/block/null_blk.c
+++ b/drivers/block/null_blk.c
@@ -79,7 +79,7 @@ MODULE_PARM_DESC(home_node, "Home node for the device");
 
 static int queue_mode = NULL_Q_MQ;
 module_param(queue_mode, int, S_IRUGO);
-MODULE_PARM_DESC(use_mq, "Use blk-mq interface (0=bio,1=rq,2=multiqueue)");
+MODULE_PARM_DESC(queue_mode, "Block interface to use (0=bio,1=rq,2=multiqueue)");
 
 static int gb = 250;
 module_param(gb, int, S_IRUGO);
-- 
1.7.1